.editorconfig

```diff
@@ -0,0 +1,14 @@
+# EditorConfig is awesome: https://EditorConfig.org
```

Comment:
Is this an editor-related file or a language-related file?

Reply:
It's just an editor-related file.

Reply:
I see. Please remove it. We should not keep any editor-related files in the repo.

Reply:
done